### PR TITLE
Changing p selectors in block 2

### PR DESCRIPTION
**Pull request by VadymVashchuk**

### CODE REVIEW

File: teamwork.css [hunk 1]
```diff
@@ -33,7 +33,7 @@ body {
   padding: 24px 48px 24px 24px;
 }
 
-p {
+.paragraph {
```
Comment (lapc002) [inline teamwork.css]: duplication of class name in one file . please merge it

File: index.html [hunk 2]
```diff
@@ -100,27 +100,27 @@ <h2 class="V-main-h2">Get started any way you&nbsp;want</h2>
       <div class="section1-part1">
         <div class="icon1"><img src="assets/images/icon-section1-part1.png" alt="icon-section1-part1"></div>
         <h3 class="semibold1"> Install via package manager </h3>
-        <p class="p-s1-p1" >
+        <p class="p-s1-p1 paragraph" >
```
Comment (lapc002) [inline index.html]: class name `paragraph` is a bad one . please choose another word for naming. In reason `p` means `paragraph` paragraph with class `paragraph`.... Too much paragraphs :D

File: teamwork.css (hunk 1 above)
Comment (VadymVashchuk) [reply teamwork.css]: done

File: index.html (hunk 2 above)
Comment (VadymVashchuk) [reply index.html]: done

File: teamwork.css [hunk 3]
```diff
@@ -33,10 +33,12 @@ body {
   padding: 24px 48px 24px 24px;
 }
 
-p {
+.text-item-in-par {
```
Comment (lapc002) [inline teamwork.css:36]: line 36 and 156 has the same className . please merge it

File: teamwork.css (hunk 1 above)
Comment (lapc002) [reply teamwork.css]: nope

File: teamwork.css (hunk 3 above)
Comment (VadymVashchuk) [reply teamwork.css:36]: 156 - it is inside @media (max-width: 1000px).
cant delete it

**Review by lapc002 — APPROVED** [review]

**Pull request merged**